cpp/examples/arrow/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@
 
 add_arrow_example(row_wise_conversion_example)
 
+add_arrow_example(rapidjson_row_converter)
```

Comment:
wrap in `if(ARROW_JSON)`?

Reply:
I'm also surprised you don't have to link to rapidjson explicitly…

Reply:
I think rapidjson is header-only. I didn't see it linked anywhere in CMakeLists for ARROW_JSON.

Reply:
Oh, good to know.

Reply:
That said, `if(ARROW_JSON)` would still be good so that this doesn't fail building if RapidJSON wasn't required for building Arrow itself.